### PR TITLE
Add all Shelby vehicles to common.js #38

### PR DESCRIPTION
**Pull request by sennahLessert**
i added shelby cars to the common.js file

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -4041,8 +4041,46 @@ const allSoloCars = {
     },
     'Mustang S281E & Mustang (NOC)': {
       'all': ['bsp'],
-    },
+    }
   },
+    'Shelby': {
+      'Corba': {
```
Comment (Bjorn248) [inline common.js:4047]: Typo here, probably meant Cobra?

File: common.js [hunk 2]
```diff
@@ -4041,8 +4041,46 @@ const allSoloCars = {
...
+          'all': ['bs']
```
Comment (Bjorn248) [inline common.js:4048]: Remove `all` here

File: common.js [hunk 3]
```diff
@@ -4041,8 +4041,46 @@ const allSoloCars = {
...
+      'GT500': {
+          '1965': ['cp'],
+          '1966': ['cp'],
+          '1967': ['fs', 'esp', 'cp'],
+          '1968': ['fs', 'esp', 'cp'],
+          '1969': ['fs', 'esp', 'cp'],
+          '1970': ['fs', 'esp', 'cp']
+      },
+      'Cobra 289': {
+          'all': ['asp']
```
Comment (Bjorn248) [inline common.js:4081]: Probably should add `bs` and `xp` to the valid classes here.

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Typo fix and removing `all` where there is also a year specified.

File: common.js (hunk 1 above)
Comment (sennahLessert) [reply common.js:4047]: yes of course ;)

File: common.js (hunk 2 above)
Comment (sennahLessert) [reply common.js:4048]: ok

File: common.js (hunk 3 above)
Comment (sennahLessert) [reply common.js:4081]: hmm i'm not sure. For Model Shelby Cobra 289 i only found "A Street Prepared" (ASP). The Model Shelby Cobra has the tags "bs" and "xp"

Comment (Bjorn248) [reply common.js:4433]: Right, the 289 is a type of Cobra. Since all Cobras are in `bs` and `xp`, so is the 289.

**Review by Bjorn248 — CHANGES_REQUESTED** [review]

**Comment by sennahLessert** [comment]
ok, i think it is ok now ;)

**Comment by Bjorn248** [comment]
Thanks!

**Pull request merged**